### DISCUSSION TITLE
Save file picker is not supported by iOS

### DISCUSSION THREAD

**Discussion by Sharkboy-j** [AvaloniaUI/Avalonia#8858] [Q&A] [answered]
Im using Avalonia 11.0.0-preview1 and trying to save file on IPad like this
```
var destinationFile =
                    await root.StorageProvider.SaveFilePickerAsync(new FilePickerSaveOptions());
```

but got Exception 

> Save file picker is not supported by iOS

Any other way to save file on IOS devices?

**Comment by maxkatz6** [comment, marked as answer]
iOS doesn't have a save file picker as a concept. Instead it has sharing API. Which is planned to be added in the future previews.

**Reply by maxkatz6** [reply]
Or you can use OpenFolderPicker and then create a file inside, if it suits your requirements.
